UnityProject/Assets/Scripts/Core/Editor/Doors/OccupationClearanceMigrator.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using Systems.Clearance;
+using Systems.Clearance.Utils;
+using NaughtyAttributes;
+using UnityEditor;
+using UnityEngine;
+
+namespace Core.Editor.Doors
+{
+	public class OccupationClearanceMigrator: EditorWindow
+	{
+		private Vector2 scrollPosition;
+		private Vector2 bigScroll;
+		private string log = "";
+
+		[SerializeField][ReorderableList][Foldout("shit")]
```

Comment:
Ah, _shit_.

Reply:
oh lmao, I was supposed to delete the script since it should only be used once